### PR TITLE
Sm/sfCommand

### PR DESCRIPTION
**Pull request by mshanemc**
> requires https://github.com/forcedotcom/salesforcedx-templates/pull/543 to correct some type inconsistency

> these are still the old names (and mostly the old messages).  They'll be moved in the next round

### What does this PR do?
migrate to sfCommand
change all FS-driven UT to NUTs
mostly use Types correctly (salesforcedx-templates is loose with how it calls yeoman and there's not much to be done from the plugin)

### What issues does this PR fix or reference?
@W-11976561@

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,49 +8,51 @@
   "dependencies": {
     "@oclif/core": "^2.0.7",
     "@salesforce/command": "^5.2.1",
-    "@salesforce/core": "^3.32.14",
-    "@salesforce/templates": "55.1.0",
+    "@salesforce/core": "^3.33.1",
+    "@salesforce/sf-plugins-core": "^2.0.0-beta.8",
```
Comment (peternhale) [inline package.json:12]: @mshanemc change to `^2.0.1`

**Review by peternhale — CHANGES_REQUESTED** [review]
I don't see schemas for command return types.

File: messages/messages.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# MissingAuraFolder
+
+Lightning bundles must have a parent folder named 'aura'.
+
+# MissingAppname
+
+Missing required flag:
+-n, --appname APPNAME name of the generated Lightning app
+See more help with --help
```
[the remark this answers is not in the document]
Comment (mshanemc) [reply messages/messages.md]: oh, yeah, I forgot I changed the tests so that they weren't asserting oclif flag errors.

File: package.json [hunk 3]
```diff
@@ -8,29 +8,31 @@
   "dependencies": {
     "@oclif/core": "^2.0.7",
     "@salesforce/command": "^5.2.1",
```
Comment (peternhale) [inline package.json:10]: Should be able to remove @salesforce/command

**Review by peternhale — CHANGES_REQUESTED** [review]

**Review by peternhale — APPROVED** [review]
@mshanemc are you going to make a pass at switch command and flag names in the tests?

**Pull request merged**